examples/tutorials/01_basics/06_sing_scales.py

```diff
@@ -16,23 +16,21 @@
 
 '''Make Cozmo sing Do Re Mi.
 
-Slight extension from hello_world.py - introduces for loops to make Cozmo "sing" the scales.
+Slight extension from the earlier 03_count.py example.
```

Comment:
I would change this docstring to be like: 

Make Cozmo "sing" the scales.
This is a minor extension to the 03_count.py example.